File: Aula_03/c++/04.cpp
#include <iostream>

bool aprovado (int nota1, int nota2){
    int media = (nota1 + nota2)/2;
    if (media < 60){
        return 0;
    }
    return 1;
}

int main(){
    int nota1, nota2;
    std::cin >> nota1 >> nota2;
    bool passou = aprovado(nota1,nota2);
    if (passou){
        std::cout << "Aprovado!\n";
    } else{
        std::cout << "Recuperacao!\n";
    }
    return 0;
}

File: Aula_03/python/04.py
def aprovado(nota1, nota2):
    media = (nota1 + nota2)/2
    if media < 60:
        return False
    return True

nota1 = int(input())
nota2 = int(input())
passou = aprovado(nota1,nota2)
if passou:
    print("Aprovado!")
else:
    print("Recuperação!")